### PR TITLE
78 offering template UI

### PR DESCRIPTION
**Pull request by landonwjohnson**
I made an example for the page at 
**app/offer/dashboard**

https://6738599a3dbeed767bab1a83-zmiirewxxa.chromatic.com/?path=/story/components-templates-layouts-offerlayout-offercontent--default

image

image

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+npx chromatic --project-token=chpt_63cf8a67c9a334f
+
```
Comment (whitagar) [inline .env:2]: Is this needed? Should this be git-ignored?

File: README.md [hunk 2]
```diff
@@ -1,21 +1,156 @@
...
+```
+
+You’ll receive a network IP address like:
+
+```bash
+  Starting Next.js on xxx.xxx.x.xxx:3000
+     ▲ Next.js 14.2.5
+     - Local:        xxx.xxx.x.xxx:3000
+     - Network:      xxx.xxx.x.xxx:3000
+```
```
Comment (whitagar) [inline README.md:69]: Oooh this is cool. I need to try this

File: README.md
Comment (whitagar) [inline README.md:1]: Love the changes here! Thanks so much for updating the README, I've been meaning to do this but hadn't gotten around to it

File: chromatic.config.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "onlyChanged": true,
+  "projectId": "Project:6738599a3dbeed767bab1a83",
+  "zip": true
+}
```
Comment (whitagar) [inline chromatic.config.json:5]: What does chromatic do?

**Review by whitagar — APPROVED** [review]
Wow! This is so thorough, I am so impressed by the initiative to do Storybook as well. Thanks for your hard work, this is awesome!

Left a couple comments but approved!

File: chromatic.config.json (hunk 3 above)
Comment (landonwjohnson) [reply chromatic.config.json:5]: I will give you access to it. it does component testing based on the stories file.

File: .env (hunk 1 above)
Comment (landonwjohnson) [reply .env:2]: uhhh yes/ I thought I did. my bad. that was for me not to forgot the token. let me change that

File: chromatic.config.json (hunk 3 above)
Comment (landonwjohnson) [reply chromatic.config.json:5]: it also makes it so the components have their own versioning system. and people can test them out without going into the web app https://6738599a3dbeed767bab1a83-zmiirewxxa.chromatic.com/?path=/story/components-templates-layouts-offerlayout-offercontent--default

**Pull request merged**